src/frontend/src/components/anchorInput.ts

```diff
@@ -37,6 +37,10 @@ export const mkAnchorInput = (
         @contextmenu=${inputFilter(isDigits, onBadInput)}
         @drop=${inputFilter(isDigits, onBadInput)}
         @focusout=${inputFilter(isDigits, onBadInput)}
+        @keypress=${onKeyPress ??
```

Comment:
doesn't lit don't bind the keypress if undefined is provided - i.e. a fallback is really needed?

Reply:
good question, will check